### DISCUSSION TITLE
V2ray中vmess节点问题

### DISCUSSION THREAD

**Discussion by Tiger-Bi** [v2fly/v2ray-core#2236] [Q&A]
为什么v2ray中的vmess节点类型能连上但是用不了 登录油管显示未连接，其他类型节点则都能用

## 你的使用场景是什么？
正常翻墙

使用除了vmess节点外的节点看youtube视频

## 你看到的异常现象是什么？
vmess节点连上用不了

直接无法访问

## 请附上你的配置

{
  "log": {
    "access": "",
    "error": "",
    "loglevel": "warning"
  },
  "inbounds": [
    {
      "tag": "socks",
      "port": 10808,
      "listen": "127.0.0.1",
      "protocol": "socks",
      "sniffing": {
        "enabled": true,
        "destOverride": [
          "http",
          "tls"
        ]
      },
      "settings": {
        "auth": "noauth",
        "udp": true,
        "allowTransparent": false
      }
    },
    {
      "tag": "http",
      "port": 10809,
      "listen": "127.0.0.1",
      "protocol": "http",
      "sniffing": {
        "enabled": true,
        "destOverride": [
          "http",
          "tls"
        ]
      },
      "settings": {
        "udp": false,
        "allowTransparent": false
      }
    }
  ],
  "outbounds": [
    {
      "tag": "proxy",
      "protocol": "shadowsocks",
      "settings": {
        "servers": [
          {
            "address": "hnt.cndns.shop",
            "method": "chacha20-ietf-poly1305",
            "ota": false,
            "password": "31347057-7af5-4562-b912-f1c217c4c160",
            "port": 27765,
            "level": 1
          }
        ]
      },
      "streamSettings": {
        "network": "tcp"
      },
      "mux": {
        "enabled": false,
        "concurrency": -1
      }
    },
    {
      "tag": "direct",
      "protocol": "freedom",
      "settings": {}
    },
    {
      "tag": "block",
      "protocol": "blackhole",
      "settings": {
        "response": {
          "type": "http"
        }
      }
    }
  ],
  "routing": {
    "domainStrategy": "IPIfNonMatch",
    "domainMatcher": "linear",
    "rules": [
      {
        "type": "field",
        "inboundTag": [
          "api"
        ],
        "outboundTag": "api",
        "enabled": true
      },
      {
        "type": "field",
        "outboundTag": "direct",
        "domain": [
          "domain:example-example.com",
          "domain:example-example2.com"
        ],
        "enabled": true
      },
      {
        "type": "field",
        "outboundTag": "block",
        "domain": [
          "geosite:category-ads-all"
        ],
        "enabled": true
      },
      {
        "type": "field",
        "outboundTag": "direct",
        "domain": [
          "geosite:cn"
        ],
        "enabled": true
      },
      {
        "type": "field",
        "outboundTag": "direct",
        "ip": [
          "geoip:private",
          "geoip:cn"
        ],
        "enabled": true
      },
      {
        "type": "field",
        "port": "0-65535",
        "outboundTag": "proxy",
        "enabled": true
      }
    ]
  }
}

**服务端配置：**

```javascript
// 在这里附上服务器端配置文件

```

**客户端配置：**

```javascript
// 在这里附上客户端配置

```

## 请附上出错时软件输出的错误日志

**服务器端错误日志：**

```javascript
// 在这里附上服务器端日志

```

**客户端错误日志：**

```javascript
// 在这里附上客户端日志

```

## 请附上访问日志

```javascript
// 在这里附上服务器端日志

```

**Comment by ququ0755** [comment]
检查服务端配置
